package.json

```diff
@@ -1,7 +1,10 @@
 {
   "name": "global-search",
   "source": "src/index.html",
-  "browserslist": "> 0.25%, not dead",
+  "browserslist": [
```

Comment:
interesting, so which plugins use this? https://github.com/browserslist/browserslist

Reply:
https://browserslist.dev/?q=PiAwLjI1JSwgbm90IGRlYWQsIGlPUyA%2BPSAxMg%3D%3D

Reply:
most bundler like parcel and webpack don't require additional plugins to read and stick to the rule of browserslist, unless need to support ie11